variants/STM32F4xx/F411V(C-E)T/variant_DISCO_F411VE.h

```diff
@@ -0,0 +1,245 @@
+#define PA2_ALT1                (PA2  | ALT1)
+#define PA2_ALT2                (PA2  | ALT2)
+#define PA3_ALT1                (PA3  | ALT1)
+#define PA3_ALT2                (PA3  | ALT2)
+#define PA4_ALT1                (PA4  | ALT1)
+#define PA5_ALT1                (PA5  | ALT1)
+#define PA6_ALT1                (PA6  | ALT1)
+#define PA7_ALT1                (PA7  | ALT1)
+#define PA7_ALT2                (PA7  | ALT2)
+#define PA7_ALT3                (PA7  | ALT3)
+#define PA15_ALT1               (PA15 | ALT1)
+#define PB0_ALT1                (PB0  | ALT1)
+#define PB0_ALT2                (PB0  | ALT2)
+#define PB1_ALT1                (PB1  | ALT1)
+#define PB1_ALT2                (PB1  | ALT2)
+#define PB3_ALT1                (PB3  | ALT1)
+#define PB4_ALT1                (PB4  | ALT1)
+#define PB5_ALT1                (PB5  | ALT1)
+#define PB8_ALT1                (PB8  | ALT1)
+#define PB9_ALT1                (PB9  | ALT1)
+#define PB14_ALT1               (PB14 | ALT1)
+#define PB14_ALT2               (PB14 | ALT2)
+#define PB15_ALT1               (PB15 | ALT1)
+#define PB15_ALT2               (PB15 | ALT2)
+#define PC0_ALT1                (PC0  | ALT1)
+#define PC0_ALT2                (PC0  | ALT2)
+#define PC1_ALT1                (PC1  | ALT1)
+#define PC1_ALT2                (PC1  | ALT2)
+#define PC2_ALT1                (PC2  | ALT1)
+#define PC2_ALT2                (PC2  | ALT2)
+#define PC3_ALT1                (PC3  | ALT1)
+#define PC3_ALT2                (PC3  | ALT2)
+#define PC4_ALT1                (PC4  | ALT1)
+#define PC5_ALT1                (PC5  | ALT1)
+#define PC6_ALT1                (PC6  | ALT1)
+#define PC7_ALT1                (PC7  | ALT1)
+#define PC8_ALT1                (PC8  | ALT1)
+#define PC9_ALT1                (PC9  | ALT1)
+#define PC10_ALT1               (PC10 | ALT1)
+#define PC11_ALT1               (PC11 | ALT1)
+
+#define NUM_DIGITAL_PINS        79
+#define NUM_ANALOG_INPUTS       16
+
+// On-board LED pin number
+#define LED_GREEN               PD12
+#define LED_BLUE                PD15
+#define LED_RED                 PD14
+#define LED_ORANGE              PD13
+#ifndef LED_BUILTIN
+  #define LED_BUILTIN           LED_GREEN
+#endif
+
+// On-board user button
+#ifndef USER_BTN
+  #define USER_BTN              PA0
+#endif
+
+// SPI Definitions
+#ifndef PIN_SPI_SS
+  #define PIN_SPI_SS            PE3
+#endif
+#ifndef PIN_SPI_SS1
+  #define PIN_SPI_SS1           PNUM_NOT_DEFINED
+#endif
+#ifndef PIN_SPI_SS2
+  #define PIN_SPI_SS2           PNUM_NOT_DEFINED
+#endif
+#ifndef PIN_SPI_SS3
+  #define PIN_SPI_SS3           PNUM_NOT_DEFINED
+#endif
+#ifndef PIN_SPI_MOSI
+  #define PIN_SPI_MOSI          PA7
+#endif
+#ifndef PIN_SPI_MISO
+  #define PIN_SPI_MISO          PA6
+#endif
+#ifndef PIN_SPI_SCK
+  #define PIN_SPI_SCK           PA5
+#endif
+
+// I2C Definitions
+#ifndef PIN_WIRE_SDA
+  #define PIN_WIRE_SDA          PB9
+#endif
+#ifndef PIN_WIRE_SCL
+  #define PIN_WIRE_SCL          PB6
+#endif
+
```

Comment:
I don't have any timer definitions.

Reply:
Use the same which are defined in the variant_generic.h:
https://github.com/stm32duino/Arduino_Core_STM32/blob/e96c3a11fb51b1d1e32120555b8863d9febb316c/variants/STM32F4xx/F411V(C-E)T/variant_generic.h#L172-L179

It is only used for Tone library.